unit_tests/connector_builder/test_property_chunking.py

```diff
@@ -196,20 +200,33 @@
 
 @freezegun.freeze_time(f"{FREEZE_DATE}T00:00:00Z")
 def test_read():
+    # We need a state time earlier than the current time otherwise we will not generate any date range
+    # partitions to process on the concurrent engine
+    day_before = (ab_datetime_parse(f"{FREEZE_DATE}T00:00:00Z") - timedelta(days=1)).strftime(
```

Comment:
just adding a comment here to highlight the above code comment.

There are a handful of these fixes in the tests which are a product of using concurrent_cursors instead legacy declarative cursors which differed in behavior